packages/sync-service/lib/electric/shape_cache/shape_status.ex

```diff
@@ -323,7 +338,7 @@ defmodule Electric.ShapeCache.ShapeStatus do
   def snapshot_started?(stack_ref, shape_handle) do
     case :ets.lookup(shape_meta_table(stack_ref), {@snapshot_started, shape_handle}) do
       [] -> false
-      [{{@snapshot_started, ^shape_handle}, true}] -> true
+      [{{@snapshot_started, ^shape_handle}, state}] -> state
```

Comment:
```suggestion
      [{{@snapshot_started, ^shape_handle}, started?}] -> started?
```
supernit - we often use `state` in genservers etc and it confuses me when I see it

Reply:
+1